css/owned-card.styl

```diff
@@ -58,8 +58,28 @@
       background: lighten(COBOLT_BLUE, 20%)
 
 .collection-card
+  -webkit-border-radius: 1em
+  border-radius: 1em
+
   .name
     margin-bottom: 1em
+    height: 2.9em
+    overflow: hidden
+    text-overflow: ellipsis
+    display: -webkit-box
+    -webkit-box-orient: vertical
+    -webkit-line-clamp: 2
```

Comment:
[nib](https://tj.github.io/nib/) will handle browser prefixing, so `box-orient` is what you want to use here.  `line-clamp` is pretty unsupported.   [A cross-browser solution](http://hackingui.com/front-end/a-pure-css-solution-for-multiline-text-truncation/) is pretty involved.  (reposted, because GitHub outdated it immediately)